### PR TITLE
Update PayPal currencies and countries

### PR DESCRIPTION
**Pull request by luigifab**
### Description

I updated PayPal currencies and countries from [Magento 2](https://github.com/magento/magento2/blob/master/app/code/Magento/Paypal/Model/Config.php).

Adding **RUB** and **INR** for `$_supportedCurrencyCodes`
Adding **CN** and **RU** for `$_supportedCountryCodes`

_I didn't searched why there are a space for each countries in `$_supportedBuyerCountryCodes`._

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

File: app/code/core/Mage/Paypal/Model/Config.php [hunk 1]
```diff
@@ -226,51 +226,379 @@ class Mage_Paypal_Model_Config
...
+    protected $_supportedCountryCodes = [
+        'AE',
+        'AR',
+        'AT',
+        'AU',
+        'BE',
+        'BG',
+        'BR',
+        'CA',
+        'CN',
```
Comment (Flyingmana) [inline app/code/core/Mage/Paypal/Model/Config.php:268]: this one (CN) is new, too?

File: app/code/core/Mage/Paypal/Model/Config.php [hunk 2]
```diff
@@ -226,51 +226,379 @@ class Mage_Paypal_Model_Config
...
+        'GF',
+        'GI',
+        'GP',
+        'GR',
+        'HK',
+        'HU',
+        'ID',
+        'IE',
+        'IL',
+        'IN',
```
Comment (Flyingmana) [inline app/code/core/Mage/Paypal/Model/Config.php:292]: was already in before

**Review by Flyingmana — APPROVED** [review]
I just commented what differs from the initial description text. But no problems as far as I see

File: app/code/core/Mage/Paypal/Model/Config.php (hunk 1 above)
Comment (luigifab) [reply app/code/core/Mage/Paypal/Model/Config.php:268]: :) it seems that I didn't check changes correctly.

File: app/code/core/Mage/Paypal/Model/Config.php (hunk 2 above)
Comment (luigifab) [reply app/code/core/Mage/Paypal/Model/Config.php:292]: :) it seems that I didn't check changes correctly.

**Pull request merged**